iocBoot/iocIpmiMgr/stIpmiMgr.cmd

```diff
@@ -1,6 +1,5 @@
 < envPaths
 epicsEnvSet("TOP", "../..")
-< IpmiMgr.config
```

Comment:
This should be in the same commit the file itself is removed, that is 639d3e7f7311751d10b61cc0a7dc9eaef0374eb6, not this one.

Reply:
This is still in "Remove install scripts" not in "Remove initialization scripts". The same for `PORT` definition.